websites/B/Baseball-Reference/presence.ts

```diff
@@ -0,0 +1,92 @@
+const presence = new Presence({
+		clientId: "1049610101725605899",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000),
+	loc = document.location;
```

Comment:
Destructure like so to improve simplicity.
```suggestion
	{ href, pathname, hostname } = document.location;
```